caasp-velero-image/caasp-velero-image.kiwi

```diff
@@ -17,7 +17,7 @@
         tag="%%PKG_VERSION%%"
         maintainer="SUSE Containers Team &lt;containers@suse.com&gt;"
         user="nobody">
-        <entrypoint execute="velero"/>
+		<entrypoint execute="/velero"/>
```

Comment:
I don't see why this is required, isn't this overwritten in the examples linked within the PR? I am not against it, just trying to understand if this is required. If our package installs velero in `/usr/bin` I'd say the expected approach is to use that by default and just add some compat link for non standard locations.

Reply:
Agree, revert this one.